CHANGELOG.md

```diff
@@ -5,12 +5,32 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Added support for PureScript 0.14 and dropped support for all previous versions (#181)
+- Renamed `Data.Array.ST.empty` to `Data.Array.ST.new` (#191, #198)
+- Renamed `group'` to `groupAll` (#194, #200)
 
 New features:
+- Added specialized monomorphic functions from `Data.Foldable` (#201):
+  - Added `foldl`, `foldr`, `foldMap`, `fold`, `intercalate` to `Array` 
+  - Added `foldl1`, `foldr1`, `foldMap1`, `foldl1`, `intercalate` to `Array.NonEmpty`
```

Comment:
Reorganized the nested bullets to cut down on duplication. What's the follow-up step for `intercalate` mentioned in the earlier discussion?

Reply:
The question is whether intercalate should get its own Array specific implementation (for performance) rather than just being a specialization of the intercalate from Data.Foldable. I think it probably should, or at least we should measure to see if an Array-based implementation would speed it up, but I suppose that doesn’t need to happen right now.

Reply:
I've opened #210 to track this.